project/.travis.yml.twig

```diff
@@ -49,6 +49,7 @@ matrix:
 {% if docs_target %}
     - php: '{{ php|last }}'
       env: TARGET=lint
+      env: lint=1
```

Comment:
We don't need this variable.

You should instead download php-cs-fixer binary on this file: https://github.com/sonata-project/dev-kit/blob/master/project/.travis/install_lint.sh (you may use the already present `composer global require`).

And add the php-cs-fixer command here: https://github.com/sonata-project/dev-kit/blob/5d31b22460395a1bc65c077f9dee1e122cc70f09/project/Makefile#L16 (BTW, you don't need the `--dry-run and the --diff` as we already check the diff at the end.

Sorry for that, I forgot to tell you.